src/app/app-winui3/InputSettingsPage.xaml.cpp

```diff
@@ -37,6 +37,17 @@ InputSettingsPage::InputSettingsPage() {
   InitializeComponent();
 }
 
+winrt::event_token InputSettingsPage::PropertyChanged(
+  winrt::Microsoft::UI::Xaml::Data::PropertyChangedEventHandler const&
+    handler) {
+  return mPropertyChangedEvent.add(handler);
+}
+
+void InputSettingsPage::PropertyChanged(
+  winrt::event_token const& token) noexcept {
+  mPropertyChangedEvent.remove(token);
+}
```

Comment:
This is needed for two-way bindings, but I think you only need one-way bindings:
- AIUI the new UI you're adding is the only way to change the new events
- if there were another way, it doesn't appear that the PropertyChanged event is emitted by anything?

Reply:
Would this matter if in future I let this be changed in a tab? e.g. having a drawing style/colour/something as a standard part of the header of a TabWithDoodles?

Reply:
If I change bindings in XAML to "OneWay" then changes in the UI don't get applied, and I return to the UI are at their previous value. I must be misunderstanding something here sorry.